assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -52,6 +57,20 @@ function InspectorProvider({ children }) {
     setInspectorContentHeight(height)
   );
 
+  useEffect(() => {
+    if (selectedElementIds.length > 0 && tab === DOCUMENT) {
+      setTab(DESIGN);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What's the complaint? Please add disable reason

Reply:
Ah, it's the missing `tab`? Yeah, that definitely needs a comment to clarify, that it's very much on purpose. Or you could create a ref to hold current tab and avoid it - that might be a cleaner solution really.

Reply:
Done!